Sources/INIParser/INIParser.swift

```diff
@@ -47,7 +47,7 @@ public final class INIParser {
         for c in line {
             switch c {
             case " ", "\t":
-                if state == .SingleQuotation || state == .DoubleQuotation {
+                if state == .SingleQuotation || state == .DoubleQuotation || state == .Title {
```

Comment:
Is this because titles can have spaces?

Reply:
Yep, config file named profiles look like `[profile foo]`, I have added reference links in `ConfigFileLoader`